include/aws/common/thread.h

```diff
@@ -49,6 +50,12 @@ struct aws_thread {
 #ifdef __cplusplus
 extern "C" {
 #endif
+
+/**
+ * Returns an instance of system default thread options.
```

Comment:
You need to specify how this is allocated, who owns cleanup, and whether it can be safely modified in-place.

Reply:
ok, I'll add,

statically allocated, copy if you want to mutate.

Reply:
Then this should be a const pointer.